image-create

```diff
@@ -207,7 +208,16 @@ try:
         memory_mb = 3072
     else:
         memory_mb = 2048
-    machine = testvm.VirtMachine(verbose=args.verbose, image=args.image, memory_mb=memory_mb, maintain=True)
+
+    console_file = None
+    if args.capture_console:
+        console_file = tempfile.NamedTemporaryFile(suffix='.log', prefix='console-')
+
+    machine = testvm.VirtMachine(verbose=args.verbose,
+                                 console_file=console_file.name,
```

Comment:
Sorry that I didn't see this earlier, as it was too distributed : This is a somewhat weird API. You stuff in a path, but the caller never actually does anything with it. Instead it's handled all internally. I think it may be cleaner to make this a `capture_console=True` flag, and generate the file name inside of VirtMachine. That also fixes the temp file scope issue that you stumbled over before.